motif/core.py

```diff
@@ -10,6 +10,9 @@
 import sox
 import tempfile as tmp
 
+from .utils import _validate_contours, _format_contour_data, _format_annotation
```

Comment:
If modules outside `utils` make use of these functions, you shouldn't prefix them with a "_"